src/test/java/it/com/atlassian/bitbucket/jenkins/internal/config/BitbucketProjectConfigurationIT.java

```diff
@@ -131,7 +130,9 @@ public void testProjectEmpty() throws IOException, SAXException {
         projectNameInput.setValueAttribute("");
         form.click();
         bbJenkinsRule.waitForBackgroundJavaScript();
-        assertNotNull(getDivByText(form, "Enter a project name"));
```

Comment:
These changes weren't needed, but it seemed a bit cleaner to assert the message on the exact div rather than just asserting that there was _some_ element with that text